api/models/graphite.go

```diff
@@ -0,0 +1,94 @@
+package models
+
+import (
+	"gopkg.in/raintank/schema.v1"
+	"sort"
+)
+
+type GraphiteRender struct {
```

Comment:
so api/models has a bunch of types defined, I think in 2/3 categories:
* incoming requests, e.g. sets of parameters that have to be validated
* output formats
* (internal only? not sure, is Req an example of this?)

perhaps there's a way to make this distinction a bit more clear? not sure how.